crnk-core/src/main/java/io/crnk/core/engine/internal/dispatcher/controller/ResourcePatchController.java

```diff
@@ -42,40 +44,54 @@ protected HttpMethod getHttpMethod() {
 
 	@Override
 	public boolean isAcceptable(JsonPath jsonPath, String method) {
```

Comment:
CrnkClient will also need to modify slightly in the ResourceRepositoryStubImpl. Because typically the semantics is /api/xy/{id} but with bulk that {id} is missing. I assume a list of ids is not really  (or having a list of ids?, but I think rather not).